bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/resources/OH-INF/thing/daikinmadoka.xml

```diff
@@ -6,14 +6,13 @@
 
 	<thing-type id="brc1h">
 		<supported-bridge-type-refs>
-			<bridge-type-ref id="dbusbluez"/>
+			<bridge-type-ref id="bluez"/>
 		</supported-bridge-type-refs>
 
 		<label>Daikin BRC1H Thermostat</label>
 		<description>A Daikin Madoka BRC1H Thermostat (BLE)</description>
 
 		<channels>
-			<channel id="rssi" typeId="rssi"/>
 			<channel id="onOffStatus" typeId="brc1h_onOffStatus"/>
```

Comment:
This change shouldn't be part of this PR.

Reply:
Oops yes ! I submitted the PR yesterday but forgot to include it.